libs/qCC_io/ShpFilter.cpp

```diff
@@ -144,6 +146,38 @@ static inline bool hasMeasurements(ESRI_SHAPE_TYPE shapeType)
 		case ESRI_SHAPE_TYPE::POLYLINE_M:
 		case ESRI_SHAPE_TYPE::POLYGON_M:
 		case ESRI_SHAPE_TYPE::MULTI_POINT_M:
+		case ESRI_SHAPE_TYPE::MULTI_PATCH:
+			return true;
+		default:
+			return false;
+	}
+}
+
+enum class ESRI_PART_TYPE: int32_t {
```

Comment:
int32_t is the default - I would remove this unless you have a special reason for it?

Reply:
Well technically `int` is the default, but...

Reply:
I chose int32_t mainly to be explicit as the PartType is stored as an int32_t in the file